src/views/AddItem.jsx

```diff
@@ -59,18 +70,25 @@ export function AddItem() {
 								id="kind_of_soon"
 								name="duration"
 								value={14}
+								onChange={(e) => setPurchaseTime(e.target.value)}
 							/>
 							<label htmlFor="kind_of_soon">Kind of Soon</label>
 						</div>
 
 						<div>
-							<input type="radio" id="not_soon" name="duration" value={30} />
+							<input
+								type="radio"
+								id="not_soon"
+								name="duration"
+								value={30}
+								onChange={(e) => setPurchaseTime(e.target.value)}
+							/>
```

Comment:
you need to add the `checked` attribute for each input as I mentioned in my example.